src/client/wrappers/loadAndErrorWrapper/index.js

```diff
@@ -16,13 +16,15 @@ const LoadAndErrorWrapper = props => {
 
   const loadingPublicProf = usersBox && usersBox.loadingPublicProf
   const unlockingProf = usersBox && usersBox.unlockingProf
+  const error = usersBox && usersBox.error
```

Comment:
Isn't there a hook for this? Has that just not been pulled into master yet?

Reply:
It was changed, but it got pulled to master only now.